test/test_transforms.py

```diff
@@ -1294,6 +1296,57 @@ def test_r3m_parallel(self, model, device):
         del transformed_env
 
 
+@pytest.mark.parametrize("device", get_available_devices())
+@pytest.mark.parametrize("model", ["resnet18", "resnet34", "resnet50"])
+class TestR3MNet:
+    @pytest.mark.parametrize("del_keys", [True, False])
+    @pytest.mark.parametrize(
+        "in_keys",
+        [["next_pixels"], ["next_pixels_1", "next_pixels_2", "next_pixels_3"]],
+    )
+    @pytest.mark.parametrize(
+        "out_keys",
+        [["next_r3m_vec"], ["next_r3m_vec_1", "next_r3m_vec_2", "next_r3m_vec_3"]],
+    )
+    def test_r3mnet_transform_observation_spec(
+        self, in_keys, out_keys, del_keys, device, model
+    ):
+        in_keys = ["next_pixels"]
```

Comment:
if `in_keys` is reset, perhaps let's remove it from the args? Or keep it in the arge but not reset it

Reply:
Sorry for the careless mistake. Fixed